contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -2,62 +2,153 @@ category: application
+        
+        Examples include:
+            * X-ray photoelectron spectroscopy (XPS)
+            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
+            * ultraviolet photoelectron spectroscopy (UPS)
+            * angle-resolved photoelectron spectroscopy (ARPES)
+            * hard X-ray photoemission spectroscopy (HAXPES)
+            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
+            * photoelectron emission microscopy (PEEM)
+            * electron spectroscopy for chemical analysis (ESCA)
+        
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
     (NXuser):
+      exists: recommended
       doc: |
         Contact information of at least the user of the instrument or the investigator
         who performed this experiment. Adding multiple users if relevant is recommended.
       name:
         doc: |
           Name of the user.
       affiliation:
-        exists: recommended
         doc: |
           Name of the affiliation of the user at the point in time when the experiment was
           performed.
-      address:
+    (NXinstrument):
+      doc: 
+       - |
+        MPES spectrometer
+       - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 12.58
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.58
+      energy_resolution(NXresolution):
         exists: recommended
-        doc: |
-          Full address (street, street number, ZIP, city, country) of the user's
-          affiliation.
-      email:
-        doc: |
-          Email address of the user.
-      orcid:
+        doc:
+        - |
+          Overall energy resolution of the MPES instrument
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.7 ff.
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.7
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.24
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.24 
+        physical_quantity:
+        type:
+        resolution(NXdata):
+          magnitude(NX_FLOAT):
+            unit: NX_ENERGY
+      device_information(NXfabrication):
         exists: recommended
-        doc: |
-          Author ID defined by https://orcid.org/.
-    (NXinstrument):
-      energy_resolution(NX_FLOAT):
-        unit: NX_ENERGY
-      (NXsource):
+        vendor:
+          exists: recommended
+        model:
+          exists: recommended
+        identifier:
+          exists: recommended
+      source_TYPE(NXsource):
+        
+        # Comment from mpes may workshop:
+        # - There is much more information which can be provided for NXsource
         doc: |
           The source used to generate the primary photons. Properties refer strictly to
```

Comment:
Since we are allowing multiple sources here, it is incorrect to say 'The source used to generate the primary photons.'

Maybe it would be better to say
```yaml
A source used to generate photons. Properties refer strictly to parameters of
the source, not of the output beam. For example, the energy of the source is
not the optical power of the beam, but the energy of the electron beam
in a synchrotron and so on.
```